python/pyspark/ml/tests.py

```diff
@@ -1255,6 +1255,17 @@ def test_output_columns(self):
         output = model.transform(df)
         self.assertEqual(output.columns, ["label", "features", "prediction"])
 
+    def test_support_for_weightCol(self):
```

Comment:
Would it make sense to also test with a classifier that doesn't have a weight col?